web/locales/en/messages.ftl

```diff
@@ -31,38 +31,45 @@ wall-of-text-more-desktop =
 show-wall-of-text = Read More
 help-us-title = Help us validate sentences!
 help-us-explain = Press play, listen & tell us: did they accurately speak the sentence below?
+
 ## ProjectStatus
-status-title = Overall project status: see how far we've come!
+status-title = Overall project status: see how far we’ve come!
 status-contribute = Contribute Your Voice
-status-loading = Loading...
-status-hours = { $hours } validated hours so far!
+status-loading = Loading…
+status-hours =
+    { $hours ->
```

Comment:
out of curiosity, is that always required? e.g. in this case it's a safe assumptions that hours is > 1

Reply:
That's not how plurals work in a lot of languages ;-)

There are [18 variants in Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals#List_of_Plural_Rules), from 0 to 6 different forms, depending on the number.

Reply:
fair enough 👍 I guess it would be syntactic sugar for cases where languages with "threshold-plurals" are used in context where a value wouldn't be under that threshold.

Reply:
I confess that I got lost in your reply :)

What I meant is: there are languages where the noun is declined differently if `$hours` is 2, 3, 4 or 6 (Welsh, for example).

en: 2 cat, 3 cats, 4 cats, 6 cats.
cy: 2 gi, 3 chi, 4 ci, 6 chi.

The only way you can support that is by using a proper plural form. The number of forms will be determined depending on the language.